src/sales-channels/asc/amazon-store-dashboard.md

```diff
@@ -3,28 +3,48 @@ title: Amazon Store Dashboard
 ---
 
 
-In the _Amazon Stores_ view on Amazon Sales Channel home page, you can click **View Store** for a store (in Active or Inactive status) to access the store's [dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
+In the _Amazon Stores_ view on Amazon Sales Channel home page, you can click **View Store** on a store card to open the store's dashboard.
 
-The store dashboard is the primary location for you to view the activity for one of your Amazon stores. With your Amazon Seller stores added and integrated, all orders and sales track through your store data view. On the dashboard, you can view revenue, track trends, and review sales data for all listings. All listings and sales are further grouped and tracked by listing type, including active, inactive, in-progress, incomplete, and orders awaiting shipment.
+The store dashboard is the primary location for you to view the activity for each of your Amazon stores. With your Amazon Seller stores added and integrated, orders and sales track through your store data view. On the dashboard, you can view revenue, track trends, and review sales data for your listings. Listings and sales are further grouped and tracked by listing type, including active, inactive, and in-progress.
 
-You can also review and update your store settings, integration settings, listing settings, price rules, order settings, and more from the store dashboard. On the dashboard, click one of your integrated stores. This store dashboard opens for stores in _Active_ and _Inactive_ status and shows data on sales, access for listings, and configuration options for the store.
-
-The store dashboard includes the following sections and provides access to additional information:
-
-- **Store Data** - Shows sales data and links to your listings by status.
-- **Revenue** - Shows a graphical representation for the store's revenue for the last 7 or 30 days.
-- **Most Recent Orders** - Shows the most recent orders that have been received from your Amazon listings association with the store.
-- **I Want to See...** - Provides links to view and update your various store, listing, rules, and order settings.
+You can also access your [store settings]({% link sales-channels/asc/ob-store-review.md %}), manage your listings, and view sales data and recent order information.
 
 ![]({% link sales-channels/asc/assets/amazon-store-dashboard.png %}){: .zoom}
 _Amazon Store Dashboard_
 
-{:.bs-callout-info}
-If you select a store with the **In Setup** status, the Onboarding steps open with populated data saved for the store integration.
+The header of the store dashboard displays the same basic store information that is shown on the store card:
+
+- Store Name
+- Magento Website
+- Status
+- Created
+- Last Updated
+
+The store dashboard also includes store data and links to settings or more information:
+
```

Comment:
I know it is hard to be consistent here, but it looks odd to have some of the named items linked and others to have links in the description. I would suggest having all links in the description.

Reply:
added link to store reports and logs, even though it requires additional text and clicks from the user.  the logs and reports have direct links in the bullet text